_posts/0000-01-02-I7L8R.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![H1](https://user-images.githubusercontent.com/107998327/178531695-82bcfd9f-3519-43a1-9938-a96230f4d22d.png)

Reply:
This is cool.